de.dlr.sc.virsat.docs.feature/src/docs/VirSat_Core_Developer_Manual.adoc

```diff
@@ -101,6 +101,57 @@ Now press the link btn:[Workbench].
 
 Done. You installed and successfully started Virtual Satellite Developer IDE.
 
+==== Git integration setup
+
+Once you have installed your Virtual Satellite Developer IDE, you need to set up your Git integration. 
+Following the next steps will help you with the progress.
+
+
+[%header,cols=">8,62a,40a"] 
+|===
+
+|Step
+|Task
+|Image
+
+|{counter:steps}.
+|Start with clicking the "Open perspective" button on the top right to open the Git perspective.
+|image::chapterIdeInstallation/subsectionGitIdeSetup/2_OpenPerspective.png[Open perspective]

```

Comment:
Let's also change the image to match the text, maybe highlight the button and show the open perspective menu